src/utils/openwhisk.js

```diff
@@ -33,6 +33,7 @@ function extractClientRequest(action) {
     path: request.params.path || '',
     extension: request.params.extension || '',
     selector: request.params.selector || '',
+    url: request.params.url || '',
```

Comment:
as mentioned in https://github.com/adobe/helix-simulator/pull/112, this should be
`path + query` since the edge never sends the url.

Reply:
now url is built on the fly